### PR TITLE
Parsers fix

### PR DESCRIPTION
**Pull request by SunweiWang**
The FindBugsXMLParser needs to be changed to work with the output produced. The reason for the change is because the output that is produced after running FindBugs is different than the result that is produced by Maven.

### CODE REVIEW

File: src/main/java/BlueTurtle/parsers/NewFindBugsXMLParser.java [hunk 1]
```diff
@@ -107,4 +80,68 @@
...
+				ArrayList<String> classPaths = ProjectInfoFinder.getClassPaths();
+				
+				String fileNWithSep = fileName.replaceAll("\\.", Matcher.quoteReplacement(File.separator)) + ".java";
+				
+				String filePath = " ";
+				
+				// Loop through all classPathes.
+				for(int i = 0; i < classPaths.size(); i++) {
+					if (classPaths.get(i).endsWith(fileNWithSep)){
+						filePath = classPaths.get(i);
```
Comment (ClintonCao) [inline src/main/java/BlueTurtle/parsers/NewFindBugsXMLParser.java]: You could stop the loop earlier if the right string is found. May save some computational time.

**Comment by ClintonCao** [comment]
Nice to see that the ProjectInfoFinder fixes the path problems with findbugs warnings.

Can we please remove one of the the parser for FindBugs? I mean I understand that it was written for the system before we added the Maven restriction, but now there we need only one parser. And having a `NewFindBugsParser` makes things more confusing. (or at least for me)

**Comment by SunweiWang** [comment]
yes, I agree, I removed `NewFindBugsParser`, and now there is only 'FindBugsParser' in the code.

**Comment by ClintonCao** [comment]
I think you should just call it FindBugsParser

**Comment by SunweiWang** [comment]
yeah, mistake in my previous comment, it has the right name in the commits.

**Comment by boninggong** [comment]
Good to see that the parser is now working again with FindBugs! Now we could filter also on FindBugs warnings in the visualizer!

**Pull request merged**